shiny/api-examples/insert_nav_panel/app-core.py

```diff
@@ -0,0 +1,77 @@
+from shiny import App, Inputs, Outputs, Session, reactive, ui
+
+app_ui = ui.page_sidebar(
+    ui.sidebar(
+        ui.input_action_button("add", "Add 'Dynamic' tab"),
+        ui.input_action_button("removeFoo", "Remove 'Foo' tabs"),
+        ui.input_action_button("addFoo", "Add New 'Foo' tab"),
+        ui.input_action_button("addTextPanel", "Add Text Panel"),
```

Comment:
No need to address this now, but FYI, we prefer `snake_case` to `camelCase` when we have the choice (i.e., the codebase isn't already heavily `camelCase` for historical reasons)